percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.25-25.12.rst

```diff
@@ -0,0 +1,61 @@
+
+New Features
+============
+
+ |Percona XtraDB Cluster| has implemented `Support for PROXY protocol <https://www.percona.com/doc/percona-server/5.6/flexibility/proxy_protocol_support.html#proxy-protocol-support>`_. The implementation is based on a patch developed by Thierry Fournier.
+
+ Variable :variable:`wsrep_dirty_reads` now has Global scope as well. (PXC-373).
+ 
+Bugs fixed 
+==========
+
+ ``SHOW STATUS LIKE ...`` and ``SHOW STATUS`` were taking time proportional to size of :variable:`gcache.size`. Bug fixed :bug:`1462674`.
+
+ When disk space would get filled with :variable:`gcache.page` files, Galera would crash when the next page file was created. Bug fixed :bug:`1488535`.
+
+ XtraBackup SST didn't clobber :file:`backup-my.cnf` which caused SST to fail. Bug fixed :bug:`1431101`.
+
+ Fixes for bug :bug:`1407770` and bug :bug:`1426378` implemented in |Percona XtraDB Cluster| :rn:`5.6.24-25.11` introduced an regression. Running ``SET GLOBAL wsrep_provider = '..';`` or ``SET GLOBAL wsrep_cluster_address = 'gcomm://';`` after setting the :variable:`wsrep_dirty_reads` could lead to server stall. Bug fixed PXC-421.
+
+ ``mysqldump`` SST could stall due to a regression introduced in |Percona XtraDB Cluster| :rn:`5.6.24
+ -25.11` by fixing the bug :bug:`1288528`. Bug fixed PXC-423.
+
+ Variable :variable:`wsrep_replicate_myisam` was not a session variable. Bug fixed PXC-332.
+
+ Missing ``datadir`` specification in :file:`my.cnf` would cause a SST failure if xtrabackup-sst method was used. Bug fixed PXC-370.
+
+ ``Percona-XtraDB-Cluster-devel-56`` package was not included in the ``Percona-XtraDB-Cluster-full-56`` metapackage on *CentOS*. Bug fixed PXC-381.
+
+ Running ``service mysql start`` and then ``service mysql@boostrap start`` afterwards would cause server shutdown. Bug fixed PXC-385.
+
+ Bug fixed PXC-390. (references https://bugs.launchpad.net/percona-xtradb-cluster/+bug/1421360 which is marked in progress and not attached to the milestone).
+
+ Bug fixed PXC-391. 
+
+ ``FLUSH LOCAL TABLES WITH READ LOCK`` command was getting stuck in |Percona XtraDB Cluster| with non existing tables which caused rsync SST to fail as well. Bug fixed PXC-399.
+
+ Bug fixed PXC-400.
+
+ |Percona XtraDB Cluster| will not blocking DDL statements on tables which are used with ``... FOR EXPORT`` or ``... WITH READ LOCK``, it will give return an error message about read lock. Bug fixed PXC-403.
+
+ Bug fixed PXC-420.
```

Comment:
Fixed the update of variable wsrep_slave_threads  wrt. default value assignment, invalid value truncation, and error issued while threads are still being closed, has been removed.